src/helpers/moneySign.js

```diff
@@ -0,0 +1,6 @@
+
+function moneySign(tv){
+    return "€" + tv.price + ",-";
```

Comment:
```suggestion
    return `€${tv.price},-`;
```
Vind ik zelf altijd wat makkelijker lezen (en uit te breiden 😉 )